src/Microsoft.Azure.WebJobs.Extensions.Kafka/Config/AzureFunctionsFileHelper.cs

```diff
@@ -11,13 +11,15 @@ namespace Microsoft.Azure.WebJobs.Extensions.Kafka
     /// <summary>
     /// Helper class for file related operations in functions running in Azure
     /// </summary>
-    internal sealed class AzureFunctionsFileHelper
+    internal static class AzureFunctionsFileHelper
```

Comment:
i see you changed the class from sealed, to static.
but then i don't see any change in how the class is being instantiated [there's only 1 file changed in this commit]

is this change still coming?

Reply:
This class was meant to be static, since the class only define static/extension methods. I can add sealed as well if you feel it is important.

Reply:
oh no, i am fine with it being static.